pyrtl/simulation.py

```diff
@@ -110,6 +110,9 @@ def _initialize(self, register_value_map=None, memory_value_map=None, default_va
                 self.memvalue[memid] = {}
 
         if memory_value_map is not None:
+            for memory in memory_value_map:
+                if isinstance(memory, RomBlock):
+                    raise PyrtlError('error, one or more of the memories in the map is a RomBlock')
```

Comment:
Please put the isinstance check inside of the other for loop
(both sim and fastsim)